### PR TITLE
PERF-5289 Ensure all high-value workloads run on all-feature-flags variants

### PR DESCRIPTION
**Pull request by BlakeIsBlake**
**Jira Ticket:** PERF-5289

**Whats Changed:**  
Self-explanatory

**Patch testing results:**  
None

**Workload Submission form:**  
None

**Related PRs:**   
None

### CODE REVIEW

File: src/workloads/query/ExpressiveQueries.yml [hunk 1]
```diff
@@ -189,6 +189,7 @@ AutoRun:
       - atlas
       - atlas-like-replica.2022-10
       - replica
+      - replica-all-feature-flags
```
Comment (jimoleary) [inline src/workloads/query/ExpressiveQueries.yml:192]: Should this also run on standalone-all-feature-flags?

**Review by jimoleary — COMMENTED** [review]
Apologies for possibly maybe making this much larger but the full list of genny HVW is [here](https://github.com/10gen/performance-tooling-docs/blob/573e9907173efa78e95d859f04bc7ee6ed1f32d9/workloads/high-value-workloads.md?plain=1#L180). 

Should these also run on a '_*-all-feature-flags_' variant?
And what's the rationale between _standalone / standalone-all-feature-flags_ v _replica / replica-all-feature-flags_ ?

**Comment by BlakeIsBlake** [comment]
The tests I included are only high-value on 3-node replica set variants. In fact, I removed ArrayTraversal since in that same high-value workload list I missed that it's only high-value on standalone, which is already included.

**Comment by BlakeIsBlake** [comment]
And I don't think where you linked is the high-value workloads, I think it's an appendix of all of the Genny workloads that Build Barons track!

**Review by jimoleary — APPROVED** [review]
LGTM

**Comment by BlakeIsBlake** [comment]
evergreen merge

**Pull request merged**